public/app/features/admin/migrate-to-cloud/onprem/EmptyState/CallToAction/CallToAction.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Box, Button, ModalsController, Text } from '@grafana/ui';
+import { Trans } from 'app/core/internationalization';
+
+import { ConnectStackDTO, useConnectStackMutation, useGetStatusQuery } from '../../../api';
+
+import { ConnectModal } from './ConnectModal';
+
+export const CallToAction = () => {
+  const [connectStack, connectResponse] = useConnectStackMutation();
+  const { isFetching } = useGetStatusQuery();
+  const onClickMigrate = async (connectStackData: ConnectStackDTO) => connectStack(connectStackData);
```

Comment:
Is this function necessary? Should be possible to use `connectStack` directly instead.